google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -155,6 +156,32 @@ public Mutation setCell(
     return this;
   }
 
+  @Override
+  public Mutation setCell(@Nonnull String familyName, @Nonnull String qualifier, long value) {
+    return setCell(familyName, wrapByteString(qualifier), value);
+  }
+
+  @Override
+  public Mutation setCell(
+      @Nonnull String familyName, @Nonnull String qualifier, long timestamp, long value) {
+
+    return setCell(familyName, wrapByteString(qualifier), timestamp, value);
+  }
+
+  @Override
+  public Mutation setCell(@Nonnull String familyName, @Nonnull ByteString qualifier, long value) {
+    long timestamp = System.currentTimeMillis() * 1_000;
```

Comment:
I would prefer to duplicate the long -> byte string conversion then duplicating the timestamp logic.

Reply:
Thanks for the suggestion, I updated setCell with same.